src/main/java/seedu/duke/task/ToDo.java

```diff
@@ -0,0 +1,55 @@
+package seedu.duke.task;
+
+public class ToDo extends Task {
+    private final String taskType = "T";
```

Comment:
I like that you have made the different task types a fixed constant across the different classes!